Signal/src/ViewControllers/DebugUI/DebugUISessionState.m

```diff
@@ -13,10 +13,19 @@
 
 @implementation DebugUISessionState
 
-+ (OWSTableSection *)sectionForContactThread:(TSContactThread *)contactThread
```

Comment:
It was _much_ simpler to give DebugUIPage a simple, common interface - even for pages that don't require a thread or require a contact thread, etc.  There's lots of ways we could model this more elegantly, but I'm trying to be pragmatic since this is just Debug UI.  So right now the responsibility falls on the creator of the page to make sure it's used appropriately.  We also use asserts.